### DISCUSSION TITLE
When is the next release for the js-sdk?

### DISCUSSION THREAD

**Discussion by dbauswel** [dapr/js-sdk#572] [General]
We're currently planning to use DAPR for a new initiative we've made a PR to add the ability to search by label. We'd like to know when the next release will be.
https://github.com/dapr/js-sdk/pull/571

**Comment by shubham1172** [comment]
Hello @dbauswel, we will release it with the next Dapr release, which you can track here https://github.com/dapr/dapr/issues/7410

**Comment by shubham1172** [comment]
It is released https://github.com/dapr/js-sdk/releases/tag/v3.3.1
